pinot-core/src/main/java/com/linkedin/pinot/core/data/manager/realtime/LLRealtimeSegmentDataManager.java

```diff
@@ -278,6 +287,15 @@ protected void consumeLoop() {
         Uninterruptibles.sleepUninterruptibly(100, TimeUnit.MILLISECONDS);
       }
     }
+
+    _serverMetrics.addMeteredTableValue(_metricKeyName, ServerMeter.ROWS_WITH_ERRORS,
```

Comment:
Yes, we want to update the metrics for row conversion status (errors, etc.) on every call of `consumeLoop()`. They won't be double-counted, since we do `_fieldExtractor.resetCounters()` at the beginning of the `consumeLoop()`